reporting/superset/Dockerfile

```diff
@@ -68,7 +68,7 @@ RUN useradd -U -m superset && \
         sqlalchemy-clickhouse==0.1.5.post0 \
         sqlalchemy-redshift==0.7.1 \
         werkzeug==0.14.1 && \
-    pip install --no-cache-dir git+https://github.com/OpenLMIS/superset-patchup.git@${SUPERSET_PATCHUP_VERSION} && \
+    pip install --no-cache-dir git+https://github.com/onaio/superset-patchup.git@${SUPERSET_PATCHUP_VERSION} && \
```

Comment:
Do we want to introduce that dependency on the ona-owned repository? @joshzamor

Reply:
@sebbrudzinski :  Yes, for now.  Lets try to keep Ona as the upstream as much as possible to avoid our copy becoming a fork.